qa/TL1_tensorflow-dali_test/test.sh

```diff
@@ -3,6 +3,9 @@ pip_packages=""
 target_dir=./docs/examples/use_cases/tensorflow/resnet-n
 
 do_once() {
+    if [ $($topdir/qa/setup_packages.py -n -u tensorflow-gpu --cuda ${CUDA_VERSION}) = -1 ]; then
```

Comment:
Similar check is in the qa/test_template.sh

Can we maybe add an arg to setup_packages.py, that returns true/false depending whether we have anything to install or not?

Reply:
In `test_template.sh` negative value is fine, `for i in `seq 0 $last_config_index`;` is just noop in this case. The only thing we do there is to make sure that `last_config_index` it not set to 0 when is negative and `one_config_only` was set.